src/components/home/HomeForm.js

```diff
@@ -0,0 +1,32 @@
+import React, {useState, useContext} from 'react';
+import {AppContext} from '../app/App';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faSearch } from '@fortawesome/free-solid-svg-icons'
+
+function HomeForm() {
+  const {dispatch} = useContext(AppContext);
```

Comment:
We have to import the AppContext from the App.js file and then we can grab dispatch